### PR TITLE
Explain MLEM simpler

### PR DESCRIPTION
**Pull request by aguschin**

### CODE REVIEW

File: profile/README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 - <img width="20px" style="vertical-align: text-top;" src="https://static.iterative.ai/logo/dvc.svg"/> DVC: Data Version Control (ML Experiments, Pipelines, Git for data) | [code](https://github.com/iterative/dvc) | [website & docs](https://dvc.org)
 - <img width="20px" style="vertical-align: text-top;" src="https://static.iterative.ai/logo/cml.svg"/> CML: Continuous Machine Learning (CI/CD for ML) | [code](https://github.com/iterative/cml) | [website & docs](https://cml.dev)
 - <img width="20px" style="vertical-align: text-top;" src="https://static.iterative.ai/logo/studio.svg"/> Iterative Studio (Collaboration for ML Projects) | [website](https://studio.iterative.ai)
-- <img width="20px" style="vertical-align: text-top;" src="https://static.iterative.ai/logo/mlem.svg"/> MLEM (Version & Deploy ML models following GitOps principles) | [code](https://github.com/iterative/mlem) | [website & docs](https://mlem.ai)
+- <img width="20px" style="vertical-align: text-top;" src="https://static.iterative.ai/logo/mlem.svg"/> MLEM (ML models deployment) | [code](https://github.com/iterative/mlem) | [website & docs](https://mlem.ai)
```
Comment (casperdcl) [inline profile/README.md]:
```suggestion
- <img width="20px" style="vertical-align: text-top;" src="https://static.iterative.ai/logo/mlem.svg"/> MLEM (ML model deployment) | [code](https://github.com/iterative/mlem) | [website & docs](https://mlem.ai)
```

File: profile/README.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 - <img width="20px" style="vertical-align: text-top;" src="https://static.iterative.ai/logo/dvc.svg"/> DVC: Data Version Control (ML Experiments, Pipelines, Git for data) | [code](https://github.com/iterative/dvc) | [website & docs](https://dvc.org)
 - <img width="20px" style="vertical-align: text-top;" src="https://static.iterative.ai/logo/cml.svg"/> CML: Continuous Machine Learning (CI/CD for ML) | [code](https://github.com/iterative/cml) | [website & docs](https://cml.dev)
 - <img width="20px" style="vertical-align: text-top;" src="https://static.iterative.ai/logo/studio.svg"/> Iterative Studio (Collaboration for ML Projects) | [website](https://studio.iterative.ai)
-- <img width="20px" style="vertical-align: text-top;" src="https://static.iterative.ai/logo/mlem.svg"/> MLEM (Version & Deploy ML models following GitOps principles) | [code](https://github.com/iterative/mlem) | [website & docs](https://mlem.ai)
+- <img width="20px" style="vertical-align: text-top;" src="https://static.iterative.ai/logo/mlem.svg"/> MLEM (ML model deployment) | [code](https://github.com/iterative/mlem) | [website & docs](https://mlem.ai)
```
Comment (shcheklein) [inline profile/README.md]: hmm, sounds like Seldon or something?

Comment (shcheklein) [reply profile/README.md]: I like the website, but it's a bit too long there and has extra explanations

may be this is indeed the best

anyways, looks like it should be "ML Model Deployment" (capital first letters)

Comment (casperdcl) [reply profile/README.md]:
```suggestion
- <img width="20px" style="vertical-align: text-top;" src="https://static.iterative.ai/logo/mlem.svg"/> MLEM (ML Model Deployment) | [code](https://github.com/iterative/mlem) | [website & docs](https://mlem.ai)
```

**Pull request merged**